app/lib/bing_api.rb

```diff
@@ -0,0 +1,54 @@
+require 'net/https'
+require 'uri'
+require 'json'
+require 'pry'
+
+class BingApiV7
+  def initialize
+    @accessKey = Rails.application.secrets.bing_api_v7
+  end
+
+  def search(company)
+
+    uri  = "https://api.cognitive.microsoft.com"
```

Comment:
move to contstants in this class URI and PATH

Reply:
ok